### ISSUE TITLE
Not able to revert lootable conversion

### ISSUE THREAD

**Issue by happy-cujo** [ironmonk88/monks-tokenbar#32]
FVTT: 0.7.9
DND5e: 1.2.4

After converting NPC to loot and trying to "Revert Lootable" the windows closes but doesn't do anything.

In the console I'm having this error:

```
Uncaught (in promise) TypeError: actor.getFlag(...) is undefined
    revertLootable https://foundry.mageroll.pl/modules/monks-tokenbar/apps/lootables.js:211
```

**Comment by ironmonk88** [comment]
That seems to indicate that the actor is either undefined or not an actor.
How comfortable with the console are you?  Could you set a break point there and tell me what's in the actor variable?

**Comment by happy-cujo** [comment]
Just to leave the note here. 

For some monsters from the DND5e SRD compendium (Bandit, Commoner, other?) revert is not working, at least on my instance. The variable passed to `revertLootable` contains in  `app.object` plain object instead of an Actor instance hence the missing getFlag. I cannot simply reinstall the system right now to be sure, but maybe SRD compendium is missing something rather that it's an issue with this module.

**Comment by ironmonk88** [comment]
I think this was an issue with certain tokens not having any items to be removed from the sheet to make it lootable.  So when it went to revert and found there were no old items, the code failed.  Should be fixed in 1.0.16.

**Comment by ironmonk88** [comment]
Closing this as 1.0.17 has just released.  Please feel free to re-open it if we havn't addressed the issue.  :-)
